apis/core/v1alpha1/featureflagconfiguration_types.go

```diff
@@ -38,15 +41,56 @@ type FeatureFlagConfigurationSpec struct {
 	// +optional
 	// +nullable
 	FlagDSpec *FlagDSpec `json:"flagDSpec"`
-	// FeatureFlagSpec is the json representation of the feature flag
-	FeatureFlagSpec string `json:"featureFlagSpec,omitempty"`
+	// FeatureFlagSpec is the json representation of the feature flag specification
+	// Deprecated: use FeatureFlagSpecV2
+	FeatureFlagSpec *string `json:"featureFlagSpec,omitempty"`
+	// FeatureFlagSpec is the structured representation of the feature flag specification
+	FeatureFlagSpecV2 *FeatureFlagSpec `json:"featureFlagSpecV2,omitempty"`
```

Comment:
It makes more sense to move this to v1valpha2 than support two versions of an API in a single resource denition